### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by Irirwanirira**
## This Pr consists of: 

- [ ] I created a method called current_user in your ApplicationController to make current user data available to all controllers.

- It will return the first user from the database.

- [ ] I created forms to perform the following functions:

- [ ] I created a Post on behalf of the current_user (use the method that you created in your ApplicationController).

- [ ] I create a comment on behalf of the current_user (use the method that you created in your ApplicationController).

- Allow Users to add likes to Posts.

### Thank you for reviewing my project

### CODE REVIEW

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Irirwanirira, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 

- [ ] Please add routes to handle the `Comment` Controller actions.

- [x] Kindly add the routes to handle `Like` Controller artion.
### **Your Currently Routes File**
routes
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: config/routes.rb [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+# root "articles#index"
+
 Rails.application.routes.draw do
+  # devise_for :users
   root "users#index"
 
   resources :users, ony: [:index, :show] do 
-    resources :posts,only: [:index, :show]
+    resources :posts,only: [:index, :show, :new, :create] do 
+      resources :comments, only: [:create]
```
Comment (codecaiine) [inline config/routes.rb]: - [x] Good job so far but  I noticed that one comment route is missing as mentioned by previous reviewer please add routes to handle the Comment Controller actions you should have something like the example below :arrow_down:  . Please fix it in order to make your project works as expected :+1: 

```
/// Already implemented    
POST   /posts/:post_id/comments(.:format)     comments#create

/// Missing route
 GET   /posts/:post_id/comments/new(.:format)     comments#new

```

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Irirwanirira , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Great form implementation :heavy_check_mark: 
- No rubocop errors :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Irirwanirira 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**